_stbt/core.py

```diff
@@ -782,7 +823,12 @@ def detect_motion(self, timeout_secs=10, noise_threshold=None, mask=None):
             imglog.imwrite("absdiff_threshold", thresholded)
             imglog.imwrite("absdiff_threshold_erode", eroded)
 
-            motion = (cv2.countNonZero(eroded) > 0)
+            out_region = _nonzero_region(eroded)
+            if out_region:
+                # Undo cv2.erode above:
+                out_region = out_region.extend(x=-3, y=-3, bottom=3, right=3)
```

Comment:
I'm not 100% certain but I think this has the possibility of drawing completely outside the frame region, say if the video cuts to black resulting in the `out_region` being the whole frame. Perhaps you could add a translucent fill to the region to compensate.